app/src/main/AndroidManifest.xml

```diff
@@ -38,5 +38,21 @@
             android:configChanges="keyboardHidden|orientation"
             android:screenOrientation="sensorLandscape"
             android:theme="@style/Theme.AppCompat.FullScreen" />
+
+        <activity
```

Comment:
We need to remove the activity part, but not sure about the fabric apikey though. For me I need the apikey to compile.

Reply:
Can this stay for now until the two forks are in sync, then go about cleaning up on xyzz's fork? I want to be able to do a clean sync back upstream.

Reply:
Sounds good to me.